src/conf.c

```diff
@@ -251,6 +251,14 @@ conf_init(conf_t *conf)
 
 	conf->interfaces->next = conf->interfaces;
 
+    /* Detect if stdin is a tty, set the default indicator to alternate.
+       Otherwise, keep it to original.*/
+    conf->alternate_output = 0;
+    if (isatty(fileno(stdin)))
+        conf->alternate_output = 1;
+    else
+        conf->alternate_output = 0;
+
```

Comment:
It should be checking `stdout`, I think, because that's where the output goes. Also, the code can be simply rewritten as:

    conf­->alternate_output = isatty(STDOUT_FILENO);

Reply:
Our target is changing the output style according to the input. So we need to check if the input is under a tty or a file. 
Here is my reference link: https://stackoverflow.com/questions/1312922/detect-if-stdin-is-a-terminal-or-pipe

Reply:
`isatty` basically returns a boolean value... why not use it directly?

Reply:
also, since we're changing the output behavior, it's `stdout` what matters.